skimage/measure/fit.py

```diff
@@ -756,18 +758,21 @@ def ransac(data, model_class, min_samples, residual_threshold,
     >>> sum(inliers) > 40
     True
 
-    Robustly estimate geometric transformation:
+    Robustly estimate geometric transformation. The argument `min_samples`
+     is an absolute number. If instead we want to use a proportion of the full
+     dataset, we can multiply by the length of the data and cast to `int`:
 
     >>> from skimage.transform import SimilarityTransform
     >>> np.random.seed(0)
     >>> src = 100 * np.random.rand(50, 2)
-    >>> model0 = SimilarityTransform(scale=0.5, rotation=1,
-    ...                              translation=(10, 20))
+    >>> model0 = SimilarityTransform(scale=0.5, rotation=1, translation=(10, 20))
     >>> dst = model0(src)
     >>> dst[0] = (10000, 10000)
     >>> dst[1] = (-100, 100)
     >>> dst[2] = (50, 50)
-    >>> model, inliers = ransac((src, dst), SimilarityTransform, 2, 10)
+    >>> get_min_samples = lambda ratio: max(1, int(ratio * len(src)))
```

Comment:
Ach. I'm as much a proponent of functional programming as the next guy (more, even), but I don't think this is friendly to Python newcomers. It's also overkill in this example since it is used only once. So I think this should just be:

```python
ratio = 0.5  # use half of the samples
sample_size = int(ratio * len(src))
```

Then use that sample size.